Update Stream Analytics module name and the picture

shijiong opened the pull request:
According to my test, the latest Azure Stream Analytics module name in this picture is the same as Stream Analytics job that we created in "Create a Stream Analytics job" section. No postfix will display. So, we need update Stream Analytics module name and the picture below.

azurecla commented:
Hi __@SSalahAhmed__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

shijiong committed:
Update Stream Analytics module name and the picture

According to my test, the latest Azure Stream Analytics module name in this picture is the same as Stream Analytics job that we created in "Create a Stream Analytics job" section. No postfix will display. So, we need update Stream Analytics module name and the picture below.
